tests/src/cli.js

```diff
@@ -56,13 +68,20 @@ describe('CLI regression tests', function () {
         this.skip();
       } else {
         if (ESLint) {
-          eslint = new ESLint({
-            useEslintrc: false,
-            overrideConfigFile: './tests/files/just-json-files/.eslintrc.json',
-            rulePaths: ['./src/rules'],
-            ignore: false,
-            plugins: { 'eslint-plugin-import': importPlugin },
-          });
+          if (semver.satisfies(eslintPkg.version, '>= 9')) {
```

Comment:
@michaelfaith it looks like this test is causing the failure in `defaults` somehow:

```
npm run mocha tests/src/cli.js tests/src/rules/default.js
```

Reply:
Oo, good find!  I see where you added a unique setting to the config for this test. Good call, and looks like that indeed fixed the issue 🎉